x11-packages/qt5-qtkeychain/ECMGeneratePriFile.cmake.patch

```diff
@@ -0,0 +1,11 @@
+--- a/cmake/Modules/ECMGeneratePriFile.cmake
++++ b/cmake/Modules/ECMGeneratePriFile.cmake
+@@ -92,7 +92,7 @@ Since pre-1.0.0.
+ set(_should_query_qt OFF)
+ if(NOT DEFINED KDE_INSTALL_USE_QT_SYS_PATHS)
+     include(ECMQueryQt)
+-    ecm_query_qt(qt_install_prefix_dir QT_INSTALL_PREFIX TRY)
++    set(qt_install_prefix_dir "/data/data/com.termux/files/usr") # idk is that correct path?
```

Comment:
This should be:
```suggestion
+    set(qt_install_prefix_dir "@TERMUX_PREFIX@")
```
As described in https://github.com/termux/termux-packages/wiki/Coding-guideline#patches